lmctl/project/handlers/brent/brent_src.py

```diff
@@ -144,6 +144,7 @@ class AnsibleLifecycleTree(files.Tree):
 
     CONFIG_DIR_NAME = 'config'
     SCRIPTS_DIR_NAME = 'scripts'
+    TEMPLATE_DIR_NAME = 'template'
```

Comment:
Added to the `AnsibleLifecycleTree`? Is this needed?

Reply:
This is typo mistake when I was merging the code line by line. Just committed the fixed code.